### PR TITLE
Allow disabling of temporary cred generation

### PR DESCRIPTION
**Pull request by ryan-norton**

### CODE REVIEW

File: doc/vaulted-edit.1.md [hunk 1]
```diff
@@ -17,3 +17,10 @@ DESCRIPTION
 Spawns an interactve mode for editing the content of an existing vault.
 
 Upon quitting, the new content is saved to the vault.
+
+AWS KEY
+-------
+
+*Substituted credentials:* Can be toggled to have your AWS credentials
```
Comment (miquella) [inline doc/vaulted-edit.1.md]: After looking at the documentation this generates, I'm less convinced this is the way to go. How about we use a bulleted list of menu options, like so:

```
 * k - Key
 * m - MFA
 * r - Role
 * c - Substitute with temporary credentials  
   Toggles whether your AWS credentials are substituted with a set of temporary
   credentials. For more details on this process, see the documentation for
   `vaulted shell`.
 * s - Show Key
 * D - Delete
```

(also, I reworded the message a little, see what you think)

File: doc/vaulted-shell.1.md [hunk 2]
```diff
@@ -16,3 +16,25 @@ DESCRIPTION
 
 Starts an interactive shell (uses the `SHELL` environment variable, if set;
 otherwise defaults to `/bin/sh`).
+
+AWS KEY
+-------
+
+By default, Vaulted substitutes your AWS credentials with a set of temporary
```
Comment (miquella) [inline doc/vaulted-shell.1.md]: If we change the verbiage elsewhere, we should also change the verbiage here.

File: doc/vaulted-edit.1.md [hunk 3]
```diff
@@ -17,3 +17,10 @@ DESCRIPTION
...
+substituted with a set of temporary credentials. For more details
+on this process, see the documentation for `vaulted shell`
```
Comment (miquella) [inline doc/vaulted-edit.1.md]: Missing a trailing period!

File: doc/vaulted-shell.1.md [hunk 4]
```diff
@@ -16,3 +16,25 @@ DESCRIPTION
...
+`vaulted edit` menu.
+
+Vaulted uses your stored credentials to generate a set of temporary STS tokens
+that remain active for the duration specified in the vault.
+
+This impacts the following environment variables:
+
+ * `AWS_ACCESS_KEY_ID`  
+   This is the temporary access key id or the access key id stored in the vault,
+   dependinding on whether substitution is enabled.
```
Comment (ryan-norton) [inline doc/vaulted-shell.1.md]: I was taking artistic liberty!

**Pull request merged**